.github/workflows/test-e2e.yml

```diff
@@ -28,3 +28,18 @@ jobs:
           echo "${KUBECONFIG_FILE}" > $HOME/.kube/config
       - name: Run e2e tests
         run: make test-e2e
+
+      - name: Notify slack on failure
+        if: failure()
+        uses: ravsamhq/notify-slack-action@v2
+        with:
+          status: ${{ job.status }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          notification_title: "E2E test failure"
+          message_format: "{emoji} *{workflow}* {status_message} in <{run_url}|{repo}>"
+          # This is the core-app user group
+          mention_groups: "C077R275SRH"
```

Comment:
question: why does the Slack pings not mention a core-app user group?

![Screenshot 2024-06-14 at 11 14 01 AM](https://github.com/celestiaorg/celestia-app/assets/3699047/09ad9364-21ab-4b88-830b-82124ab4c7c3)

Reply:
actually i'm not sure. do we want to be explicitly mentioned? we're gonna get notified anyway and it's only our team in that channel.

Reply:
IMO we don't need to be explicitly mentioned so these line scan prob be removed b/c it doesn't look like they work right now

Reply:
fixing it now!